addon/lib/metrics.js

```diff
@@ -66,7 +67,7 @@ function makeTimestamp(time) {
 const Metrics = module.exports = {
 
   init: function() {
-    store.browserLoadedTimestamp = Date.now();
+    store.browserLoadedTimestamp = Services.startup.getStartupInfo().process;
```

Comment:
We should probably just not set this, and instead replace `store.browserLoadedTimestamp`'s use in `makeTimestamp` a few lines above.